extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -169,7 +169,7 @@ oppia.directive('imageWithRegionsEditor', [
             if (newVal !== '') {
               // Loads the image in hanging <img> tag so as to get the
               // width and height.
-              $('<img/>').attr('src', $scope.getPreviewUrl(newVal)).on(
+              $('<img/>').attr('src', $scope.getPreviewUrl(newVal.filename)).on(
```

Comment:
Since we already have the width and height we don't need to use the `$('<img/>')`, you can just `$scope.originalImageWidth =  newVal.width`

Reply:
Done!